File: Uri Online Judge/Unforeseen at Christmas.cpp
#include<cstdio>

int main(){
    int alumnos,computadores,quemados,sincompilador;
    scanf("%d %d %d %d",&alumnos,&computadores,&quemados,&sincompilador);

    int malos = quemados+sincompilador;
    alumnos++;

    if(alumnos > computadores-malos){
        if(quemados > sincompilador) printf("Caio, a culpa eh sua!\n");
        else printf("Igor bolado!\n");
    }else printf("Igor feliz!\n");


    return 0;
}

File: Uri Online Judge/Unforeseen at Christmas.py
from sys import stdin

datos = [int(x) for x in stdin.readline().split()]

datos[0] += 1

if datos[0] <= datos[1]-datos[2]-datos[3]: print("Igor feliz!")
elif datos[2] > (datos[2]+datos[3])/2: print("Caio, a culpa eh sua!")
else: print("Igor bolado!")
